pegasus/sites.v3/hourofcode.com/public/events/2015/index.haml

```diff
@@ -0,0 +1,40 @@
+---
+title: <%= hoc_s(:events_all_title) %>
+layout: wide
+nav: events_nav
+---
+-SOLR = Solr::Server.new(host:CDO.solr_server)
+
+-by_country = {}
+-by_state = {}
+-country_names = {}
+-state_names = {}
+
+-events = SOLR.query(q:"*:*", fq:["kind_s:HocSignup2015", "location_country_s:[* TO *]", "location_country_code_s:[* TO *]", "organization_name_s:[* TO *]"], sort:'location_country_s asc, location_state_code_s asc', rows:1000)
```

Comment:
I guess wearing my MVC hat it would be good to get some of this code out of a view and into a controller or model at some stage.